google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/BitReversedSequenceStyleGenerator.java

```diff
@@ -203,46 +213,6 @@ void configureExcludedRanges(String sequenceName, Properties params) throws Mapp
     this.excludeRanges = parseExcludedRanges(sequenceName, params);
   }
 
-  @VisibleForTesting
```

Comment:
This method is moved to the `EnhancedBitReversedSequenceStyleGenerator`, as both generators can share this code. It makes more sense to have it in the non-deprecated class than in this one.